scripts/ci/run-jenkins.sh

```diff
@@ -352,7 +352,7 @@ if [[ ${CI_TAGS} == *'webassembly'* ]] || [[ ${CI_TAGS} == *'wasm'* ]];
             ${TESTCMD} --label=mini --timeout=20m $gnumake -C sdks/wasm run-all-mini
             ${TESTCMD} --label=v8-corlib --timeout=20m $gnu$gnumake -C sdks/wasm run-v8-corlib
             # https://github.com/mono/mono/issues/19957
-            #${TESTCMD} --label=debugger --timeout=40m $gnumake -C sdks/wasm run-debugger-tests
+            ${TESTCMD} --label=debugger --timeout=40m $gnumake -C sdks/wasm run-debugger-tests
             ${TESTCMD} --label=mini-system --timeout=60m $gnu$gnumake -C sdks/wasm run-all-System
```

Comment:
This was disabled because of a browser crash, does this really fix it ?

Reply:
No no, I just enabled to test this PR, I will disable before merge